metaflow/plugins/aip/aip.py

```diff
@@ -957,7 +957,9 @@ def _create_volume(
         size: str,
         volume_type: Optional[str],
     ) -> Tuple[ResourceOp, PipelineVolume]:
-        volume_name = "{{pod.name}}"
+        # AIP-8595(talebz): to avoid volume name "must be no more than 63 characters error"
+        # truncate from the beginning (the suffix has more entropy)
+        volume_name = "{{=sprig.trunc(-63, pod.name)}}"
```

Comment:
Will this cause duplicated volume names in a for each loop, where pods mostly differ by suffix?

Reply:
Never mind. I see it's from the end using `-` sign.